apps/ledger-live-desktop/src/renderer/families/evm/StakeFlowModal/EthStakingModalBody.tsx

```diff
@@ -2,23 +2,31 @@ import { Flex, Text } from "@ledgerhq/react-ui";
 import { Account } from "@ledgerhq/types-live";
 import React, { useCallback, useState } from "react";
 import { useHistory } from "react-router-dom";
+import BigNumber from "bignumber.js";
+import { useTranslation } from "react-i18next";
 
 import { appendQueryParamsToDappURL } from "@ledgerhq/live-common/platform/utils/appendQueryParamsToDappURL";
-import { useTranslation } from "react-i18next";
+import { LiveAppManifest } from "@ledgerhq/live-common/platform/types";
 import { track } from "~/renderer/analytics/segment";
 import CheckBox from "~/renderer/components/CheckBox";
 import EthStakeIllustration from "~/renderer/icons/EthStakeIllustration";
-import { openURL } from "~/renderer/linking";
+
 import {
   CheckBoxContainer,
   LOCAL_STORAGE_KEY_PREFIX,
 } from "~/renderer/modals/Receive/steps/StepReceiveStakingFlow";
 import { ListProvider, ListProviders } from "./types";
 import { getTrackProperties } from "./utils/getTrackProperties";
 
-import { LiveAppManifest } from "@ledgerhq/live-common/platform/types";
 import ProviderItem from "./component/ProviderItem";
 
+const MAGNITUDE = 18;
+// Scale Eth to 18 decimals
+const eth = (n: number | BigNumber) => BigNumber(n).times(BigNumber(10).pow(MAGNITUDE));
```

Comment:
This can just be a const:
```suggestion
const ETH_LIMIT = BigNumber(32).times(BigNumber(10).pow(18));
```

Reply:
nice one thanks